charts/pwa/README.md

```diff
@@ -55,6 +55,25 @@ Both `cacheIgnoreParams` and `multiChannel` parameters take precedence over any
 | `hybrid.backend.service` | ICM Web Adapter service name                 | `icm-web`     |
 | `hybrid.backend.port`    | ICM Web Adapter service port                 | `443`         |
 
+## Shared Redis Cache
+
+> [!IMPORTANT]
+> The shared Redis cache for the nginx containers requires Intershop PWA version 5.0.0 or newer.
+
+The PWA Helm chart supports a shared Redis cache for the nginx containers. To enable it, add the following configuration to your values file:
+
+```yaml
+redis:
+  uri: rediss://user:password@redis.cloud.com:6379
```

Comment:
rediss

Reply:
Yes. Redis secure scheme. 😅

Examples https://github.com/lettuce-io/lettuce-core/wiki/Redis-URI-and-connection-details#uri-syntax